tests/test_mks_client_nnp_hr.py

```diff
@@ -0,0 +1,63 @@
+import os
+from datetime import date
+
+FIXTURE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'fixtures')
+NNP_RESPONSE_PATH = os.path.join(FIXTURE_PATH, "hr_nnp_response.xml")
+
+NNP_HR_RESPONSE = {
```

Comment:
Ik zet dit altijd in de return van een functie, dan krijg je namelijk altijd een nieuw dict ipv een referentie naar deze ene.

mocht je wat data willen aanpassen in een test, heb je geen side effects van andere tests.